scripts/publish_layers.sh

```diff
@@ -13,7 +13,8 @@ set -e
 RUBY_VERSIONS_FOR_AWS_CLI=("ruby2.5" "ruby2.7")
 LAYER_PATHS=(".layers/datadog-lambda_ruby2.5.zip" ".layers/datadog-lambda_ruby2.7.zip")
 LAYER_NAMES=("Datadog-Ruby2-5" "Datadog-Ruby2-7")
-AVAILABLE_REGIONS=(us-east-2 us-east-1 us-west-1 us-west-2 ap-east-1 ap-south-1 ap-northeast-2 ap-southeast-1 ap-southeast-2 ap-northeast-1 ca-central-1 eu-north-1 eu-central-1 eu-west-1 eu-west-2 eu-west-3 sa-east-1)
+AVAILABLE_REGIONS=$(aws ec2 describe-regions | jq -r '.[] | .[] | .RegionName')
```

Comment:
Do we maybe want to avoid making this call if $1 is specified?

Reply:
There's actually some validation later in the code to ensure $1 is one the regions returned here so we have to get this list regardless.